### PR TITLE
Improved user/dev usability

### PR DESCRIPTION
**Pull request by PabloL007**
This commit introduces several minor improvements like using empty arrays in API responses over null fields or typo corrections in the README, as well as, a new file collector that allows users to easily import csv-type data into nerd. The focus is to make it a bit easier to use and understand both for people interested in adding it to their stack and those that want to contribute. Finally, I've added some issue templates.

### CODE REVIEW

File: internal/series/series.go [hunk 1]
```diff
@@ -71,7 +72,7 @@ func ProcessUpdate(event event.Event, ps pointstores.PointStore, tServ chan type
 			}
 		}
 		// Queue up training if enough points are available
-		req, err := ml.Required(len(inputs), 1, conf.ML.HLayers, conf) // 1 because we'll be creating individual nets for each output
+		req, _ := ml.Required(len(inputs), 1, conf.ML.HLayers, conf) // 1 because we'll be creating individual nets for each output
```
Comment (PabloL007) [inline internal/series/series.go:75]: At the moment, the only reason for err to not be nil would be for an invalid net type to be configured. But it doesn't make sense to show an error about it this at this point in the execution, that is why I've opted to add a check in the config module and discard the value here instead.

File: api/series.go [hunk 2]
```diff
@@ -94,18 +65,18 @@ func (h *Handler) ListPoints(c *gin.Context) {
 	exists, err := h.PS.Exists(id)
 	if err != nil {
 		logger.Error("Failed to check if series with id "+id+" exists", err)
-		c.JSON(http.StatusInternalServerError, types.APIError{Msg: "Error fetching points, see logs for more info"})
+		c.JSON(http.StatusInternalServerError, types.NewErrorRes("Error fetching points, see logs for more info"))
 		return
 	}
 	if !exists {
-		c.JSON(http.StatusNotFound, types.APIError{Msg: "Series with id " + id + " could not be found"})
+		c.JSON(http.StatusNotFound, types.NewErrorRes("Series with id "+id+" could not be found"))
```
Comment (mafdezhdez) [inline api/series.go]: 'with the ID'?

Comment (PabloL007) [reply api/series.go]: Yeah, I missed those 3, fixed and pushed, thanks

File: cmd/nerd/main.go [hunk 3]
```diff
@@ -47,7 +47,7 @@ func main() {
 	api, err := api.New(tServ, *conf)
 	if err != nil {
 		logger.Error("Error encountered initializing API", err)
-		return
+		os.Exit(1)
 	}
 	srv := &http.Server{
 		Addr:    ":5400",
```
Comment (mafdezhdez) [inline cmd/nerd/main.go:53]: I think in line 69, you should add, after the logger.Error(...), an os.Exit(1)

Comment (PabloL007) [reply cmd/nerd/main.go:53]: Fixed and pushed

**Review by mafdezhdez — APPROVED** [review]

**Pull request merged**